ReactiveCocoa/Swift/Property.swift

```diff
@@ -95,6 +95,57 @@ extension MutableProperty: SinkType {
 	}
 }
 
+/// Wraps a `dynamic` property, or one defined in Objective-C, using Key-Value
+/// Coding and Key-Value Observing.
+///
+/// Use this class only as a last resort! `MutableProperty` is generally better
+/// unless KVC/KVO is required by the API you're using (for example,
+/// `NSOperation`).
+@objc public final class DynamicProperty: NSObject, PropertyType {
+	public typealias Value = AnyObject?
```

Comment:
Yeah, that's fair :)

I know that whenever I use this, I'll definitely implement in my app the generic version with the forced cast though :D